src/defargs.l

```diff
@@ -140,6 +141,9 @@ CPPC  "/\/"
 %x	FuncQual
 %x	ReadDocBlock
 %x	ReadDocLine
+%x  ReadAnnotation
+%x  ReadStringLiteral
+%x  ReadEscape
```

Comment:
Sorry for the misalignment; did you notice the blank between ``%x`` and the identifier above my edits is actually a tab, which is why it looked good in my editor. Would it be okay if I aligned the whole block consistently? Or was there a reason for the original misalignment and for using the tabs?

Reply:
In principle there should not be any tabs in the code but they are a remainder of the past (same accounts for white space at the end of a line, shouldn't be present). In a lot of files this has already been corrected by @doxygen with a special procedure. Unfortunately it is still present in a number of files (and giving headaches over and over again due to inconsistent tab settings, especially in Python where whitespace has, unfortunately, a special meaning for defining blocks).

Reply:
Turns out there are still *plenty* of tabs in ``defargs.l``. I think replacing all of them would pretty much contaminate the changeset of this PR and should be a separate effort. So, for now I just fixed the alignment of the lex declaration block.

Reply:
Correct, the big change should be a separate changeset and not be part of a "real" fix.